### ISSUE TITLE
Mu ignores English unit setting

### ISSUE THREAD

**Issue by thoraxe** [patrickmoore/Mu#5]
I manually edited the Mu config file (Mu.exe.config) to force units to English:

```xml
            <setting name="Units" serializeAs="String">
                <value>1</value>
            </setting>
            <setting name="Exporter" serializeAs="String">
                <value>Csv</value>
            </setting>
```
Mu says Units:English

However, the CSV file created does not have English units. Ex:

LFpressure | AirPressure
-- | --
kPa | Hg

**Comment by patrickmoore** [comment]
I'll see about adding the conversion to csv's.

**Comment by thoraxe** [comment]
Oh - is that only working for MoTeC right now?

That'd be great. I can deal with it for now (I'm building an analyzer app).

**Comment by patrickmoore** [comment]
Yeah. The csv exporting is something that took me about an hour to put together.  I think I didn't add it initially because I currently don't have an explicit mapping between Metric and English units.  In the Motec exporter, the corresponding English unit is defined for the channel.

**Comment by patrickmoore** [comment]
Fixed with 1.9.0.0
